File: scratch/check_html.php
<?php
$html = file_get_contents('http://localhost:8000/case-studies/sellvixa.php');
if ($html === false) {
    echo "Error: Could not fetch page.\n";
    exit(1);
}
$lines = explode("\n", $html);
foreach ($lines as $i => $line) {
    if (strpos($line, 'folded-tab') !== false) {
        echo ($i + 1) . ": " . trim($line) . "\n";
    }
}
?>

File: scratch/check_html.py
import urllib.request

try:
    url = "http://localhost:8000/case-studies/sellvixa.php"
    response = urllib.request.urlopen(url)
    html = response.read().decode('utf-8')
    lines = html.splitlines()
    for i, line in enumerate(lines):
        if "folded-tab" in line:
            print(f"Line {i+1}: {line}")
except Exception as e:
    print(f"Error: {e}")
